bindings/c/fdb_c.cpp

```diff
@@ -509,8 +509,13 @@ FDBFuture* fdb_transaction_get_range_impl(
 
 	if (target_bytes == GetRangeLimits::BYTE_LIMIT_UNLIMITED)
 		target_bytes = mode_bytes;
-	else if (mode_bytes != GetRangeLimits::BYTE_LIMIT_UNLIMITED)
-		target_bytes = std::min(target_bytes, mode_bytes);
+
+	if(target_bytes > 131072){
```

Comment:
My initial gut says that this shouldn't change, and rather the Java bindings should use the exact streaming mode with their desired byte limit when specifying a custom strategy. That would bypass the existing check that was deleted, and If you want add a cap for target bytes, then it should be done at the Java level.

Otherwise, this is a behavior change for other users of the exact streaming mode.